src/lib/mina_base/user_command.ml

```diff
@@ -190,13 +190,15 @@ let application_nonce (t : t) =
   | Parties p ->
       Parties.application_nonce p
 
+let expected_target_nonce t = Account.Nonce.succ (application_nonce t)
```

Comment:
Should we rename `application_nonce` to `applicable_at_nonce`? I was initially confused, thinking it was something to do with a zkApp application.